WebAppServiceBus/WebAppServiceBus/ReceivedMessageStore.cs

```diff
@@ -1,38 +1,53 @@
-﻿using Microsoft.Azure.ServiceBus;
-using Microsoft.Azure.ServiceBus.Primitives;
+﻿using Azure.Messaging.ServiceBus;
 using System.Collections.Generic;
-using System.Text;
+using System;
+using Azure.Identity;
 using System.Threading.Tasks;
 using WebAppServiceBus.Models;
 
 namespace WebAppServiceBus
 {
     public static class ReceivedMessageStore
     {
-        private static QueueClient _receiveClient = null;
         private static List<string> _receivedMessages = new List<string>();
 
-        public static void Initialize(ServiceBusConfiguration config)
+        public static async Task InitializeAsync(ServiceBusConfiguration config)
```

Comment:
I think we can pass the ServiceBusClient in here instead of using a separate one.